freqtrade/optimize/hyperopt.py

```diff
@@ -228,13 +235,13 @@ def format_results(self, results: DataFrame) -> str:
         Return the format result in a string
         """
         trades = len(results.index)
-        avg_profit = results.profit_percent.mean() * 100.0
+        avg_profit = results.profit_percent.mean()
```

Comment:
Are you sure that this is correct?

`profit_percent` is calculated using `calc_profit_percent()` in `persistence.py` - which does not seem to calculate, but a factor.

Rest looks good :+1: